src/Microsoft.OData.Edm/Schema/Interfaces/IEdmOptionalParameter.cs

```diff
@@ -0,0 +1,21 @@
+//---------------------------------------------------------------------
+// <copyright file="IEdmOptionalParameter.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using Microsoft.OData.Edm.Vocabularies;
+
+namespace Microsoft.OData.Edm
+{
+    /// <summary>
+    /// Represents an optional parameter of an EDM operation.
+    /// </summary>
+    public interface IEdmOptionalParameter : IEdmOperationParameter
+    {
+        /// <summary>
+        /// Gets the default value of this property.
+        /// </summary>
+        string DefaultValueString { get; }

```

Comment:
Yep; we use "DefaultValueString" to reinforce that it is the string representation of the default value (i.e., "1", not 1).

---
In reply to: [123295172](https://github.com/OData/odata.net/pull/828#discussion_r123295172) [](ancestors = 123295172,123293308)